Games/2048/Game.h

```diff
@@ -30,7 +30,7 @@
 
 class Game final {
 public:
-    Game(size_t);
+    Game(size_t board_size, size_t target_score = 0);
```

Comment:
This accepts target *score*, but the implementation accepts target *tile*.

Reply:
Oh, I missed that
Thanks for catching it~